### ISSUE TITLE
Add to MavenCentral repository

### ISSUE THREAD

**Issue by GreyLabsDev** [barteksc/AndroidPdfViewer#1049]
Please add stable version to MavenCentral repo, because jcenter will be stopped
[JCenter Shutdown - Gradle blog](https://blog.gradle.org/jcenter-shutdown
)

**Comment by mksoftware101** [comment]
Yes, it is in jitpack: https://jitpack.io/#barteksc/AndroidPdfViewer/Tag

**Comment by PetrHula** [comment]
JitPack provides old version only -> 3.1.0-beta1. We need 3.2.0-beta2 on MavenCentral od JitPack, please...

**Comment by srisindhusaride** [comment]
Now that JCenter is stopped and you cant download dependencies from JCenter, what can I do here:
I removed JCenter and have this in my gradle file:    implementation 'com.github.barteksc:AndroidPdfViewer:3.1.0-beta.1'

But i still get this error: Received status code 502 from server: Bad Gateway
